spec/system/notifications/notifications_page_spec.rb

```diff
@@ -47,7 +47,7 @@ def validate_reply(id)
 
   it "allows user to follow other users back" do
     follow = leslie.follow(alex)
-    Notification.send_new_follower_notification_without_delay(follow, "Published")
```

Comment:
Huh, I wonder why this was `"Published"` before.

Reply:
ahahaha me too :D I guess it worked because Rails transformed the string to its booleanness thruthy? :D

Reply:
Ohhh hahaha yeah probably!